### ISSUE TITLE
Generep steals money :)

### ISSUE THREAD

**Issue by hinch** [NCC-Lykos/neocron-classic#335]
This is a semi complicated bug to replicate there's a few processes involved.

- Register genereps until you have a scrollable page of genereps in your list.
- Once you have this right click generep to use as usual.
- Select a generep at the top of the list on page 1 and ensure that it says "cost" as 4000nc as you would expect.
- Now without clicking on any other genereps in that list use the mouse scroll wheel to move to page 2 ensure that the selected generep is pushed off the top of the list view.
- "cost" should now read 500nc and clicking the use button instead of teleporting you to your selected destination - yes the previously selected one is still selected just off screen - it just takes your money from you and you stay where you are.
- Scroll back up so your previously selected rep is now visible and the price should revert back to 4000nc and clicking teleport should correctly take the 4k and move you to your desired location.

**Comment by ghost** [comment]
Interesting find, @hinch !

**Comment by hinch** [comment]
@NCC-Gnisha  no shit. you been able to replicate it?

**Comment by ghost** [comment]
Not yet. I still need to run around and grab some GRs to fill up my list some. Been busy with work this week.

**Comment by NCC-Therenall** [comment]
I fixed this in a patch that will be deployed in a few minutes. It also no longer matters where you are in the scroll list, the GR you have selected will be the one displayed for warp and will be the one you are charged for and warped too. I also fixed the activate GR and warp buttons not always consistently working. 

That being said, the death respawn window is a mess and half the crap doesn't work on that. I'll get a fix for that one in a few days, but going to close this for now.
